quinn/src/lib.rs

```diff
@@ -672,83 +559,101 @@ pub type Incoming = mpsc::Receiver<NewConnection>;
 
 /// A connection initiated by a remote client.
 pub struct NewConnection {
+    /// The driver for the connection
```

Comment:
Worth having a hint that this is a future that should be spawned immediately here.

Reply:
Done (merged `NewClientConnection` and `NewConnection`, since they seem identical now).